creator-node/src/routes/files.js

```diff
@@ -289,61 +367,70 @@ const getCID = async (req, res) => {
         time: `${Date.now() - startMs}ms`
       })
     }
-  } else {
-    // Check if CID record is in DB, error if not
-    startMs = Date.now()
-    try {
-      const queryResults = await models.File.findOne({
-        where: {
-          multihash: CID
-        },
-        order: [['clock', 'DESC']]
-      })
+  }
+
+  /**
+   * If not found on FS, check if CID record is in DB, error if not found
+   */
+  startMs = Date.now()
+  try {
+    const queryResults = await models.File.findOne({
+      where: {
+        multihash: CID
+      },
+      order: [['clock', 'DESC']]
+    })
+    decisionTree.push({
+      stage: `DB_CID_QUERY`,
+      time: `${Date.now() - startMs}ms`
+    })
+
+    if (!queryResults) {
       decisionTree.push({
-        stage: `DB_CID_QUERY`,
-        time: `${Date.now() - startMs}ms`
+        stage: `DB_CID_QUERY_CID_NOT_FOUND`
       })
-
-      if (!queryResults) {
-        decisionTree.push({
-          stage: `DB_CID_QUERY_CID_NOT_FOUND`
-        })
-        logGetCIDDecisionTree(decisionTree, req)
-        return sendResponse(
-          req,
-          res,
-          errorResponseNotFound(
-            `${logPrefix} No valid file found for provided CID`
-          )
-        )
-      } else if (queryResults.type === 'dir') {
-        decisionTree.push({
-          stage: `DB_CID_QUERY_CONFIRMED_DIR`
-        })
-        logGetCIDDecisionTree(decisionTree, req)
-        return sendResponse(
-          req,
-          res,
-          errorResponseBadRequest('this dag node is a directory')
+      logGetCIDDecisionTree(decisionTree, req)
+      return sendResponse(
+        req,
+        res,
+        errorResponseNotFound(
+          `${logPrefix} No valid file found for provided CID`
         )
-      } else {
-        decisionTree.push({
-          stage: `DB_CID_QUERY_CID_FOUND`
-        })
-      }
-    } catch (e) {
+      )
+    } else if (queryResults.type === 'dir') {
       decisionTree.push({
-        stage: `DB_CID_QUERY_ERROR`,
-        time: `${Date.now() - startMs}ms`,
-        error: `${e.message}`
+        stage: `DB_CID_QUERY_CONFIRMED_DIR`
       })
       logGetCIDDecisionTree(decisionTree, req)
       return sendResponse(
         req,
         res,
-        errorResponseServerError(`${logPrefix} DB query failed`)
+        errorResponseBadRequest('this dag node is a directory')
       )
+    } else {
+      decisionTree.push({
+        stage: `DB_CID_QUERY_CID_FOUND`
+      })
+      // This should never happen, logging in case it does
+      if (storagePath !== queryResults.storagePath) {
```

Comment:
can we add these values to the decision tree instead? I rather we dont add conditional logs if we can avoid it. so the decision tree can look something like 
```
      decisionTree.push({
        stage: `DB_CID_QUERY_CID_FOUND`,
        expectedStoragePath: storagePath,
        dbStoragePath: queryResults.storagePath
      })
```

Reply:
this should never ever happen so i explicitly added the log bc i want to surface it if it does

Reply:
yeah, this is fine. I just rather avoid conditional logging + adding code for a case where it looks like itll never hit. we can maybe consider something like this where an undefined value will not be logged.

```
decisionTree.push({ stage: `DB_CID_QUERY_CID_FOUND`, storagePathsMatch: storagePath !== queryResults.storagePath ? true : undefined })
```